### PR TITLE
Feature/sle 4237 hana sr wizard

### PR DESCRIPTION
**Pull request by sknorr**
I accidentally closed #7 , when I deleted the develop branch and moved to master. Reopening.

### CODE REVIEW

**Comment by tomschr** [comment]
Here is a screenshot about this little section:

SAP-Unattended-Setup

**Comment by diegoakechi** [comment]
@tomschr Some clarifications here: the `readconfig` and the `unattended` options can be used independently, they are not related steps to each other.

With the `readconfig` only, the module will parse the file and present the Summary Screen of the module, for the user validation. In tha case, the module will only perform any action if the user click on the button for that.

The second option, with `unattended` included, basically the module will parse the file and start the installation, without any user intervention.

The fact that these options were included in a list gives me the impression they are sequential steps on the module usage.

File: xml/s4s_setup_cluster.xml [hunk 1]
```diff
@@ -17,6 +17,12 @@
   wizard once and can then re-apply the saved configuration without
   user interaction.
  </para>
+ <para>
+  Administrators can now use the &hana;-SR Wizard to configure the &hana;
+  cluster on Azure and only have to define a configuration inside the
```
Comment (diegoakechi) [inline xml/s4s_setup_cluster.xml]: @sknorr @tomschr  Maybe here is needed to clarify a little bit. We have 2 new features, one is the capability of running the module unattended. It applies to any usage, like the most common that is the on-premisses deployments.

The second feature is the Cluster Configuration on Azure, that was implemented and the YaST module identifies automatically when  running on Azure, and configures an extra resource needed on Pacemaker, that is a Netcat service to provide healthprobe for the Azure Load Balancer.

File: xml/s4s_setup_cluster.xml [hunk 2]
```diff
@@ -657,11 +663,12 @@
  </sect1>
  <sect1 xml:id="sec-s4s-hana-cluster-wizard-semiautomatic">
   <title>Unattended Setup Using &hana;-SR Wizard</title>
-  
+
   <para>
    To deploy a preconfigured &hana;-SR environments, &suse; offers an
    unattended installation mode for the &hana;-SR wizard.
    To use it, proceed as follows:
+   <remark>sknorr 2019-05-29: mention where I need to perform these steps (what machine).</remark>
```
Comment (diegoakechi) [inline xml/s4s_setup_cluster.xml]: Is need to perform the SAP HANA installation on both nodes.

File: xml/s4s_setup_cluster.xml [hunk 3]
```diff
@@ -674,6 +681,7 @@
    <step>
     <para>
      Save the configuration on its last screen.
+     <remark>sknorr 2019-05-29: how?</remark>
```
Comment (diegoakechi) [inline xml/s4s_setup_cluster.xml]: In the last screen there is a button to save the configuration.

File: xml/s4s_setup_cluster.xml [hunk 4]
```diff
@@ -685,16 +693,12 @@
    <step>
     <para>
      Import, validate, and install the cluster unattendedly,
+     <remark>sknorr 2019-05-29: Do I need to switch to a different machine now?</remark>
```
Comment (diegoakechi) [inline xml/s4s_setup_cluster.xml]: The process needs to be started on the machine that will be the Primary HANA, as a consequence, the other machine will become the secondary on the HANA System Replication configuration.

File: xml/s4s_setup_cluster.xml (hunk 2 above)
Comment (sknorr) [reply xml/s4s_setup_cluster.xml]: Thanks, @dakechi, but going by the steps:

* Step 1 & 2, running the wizard -- would you do that on some kind of local machine/test machine? Does that machine need a complete HANA installation?

* Step 3 & 4 -- as far as I understand, need to be done on each of the two production machines. Is that correct?

File: xml/s4s_setup_cluster.xml (hunk 3 above)
Comment (sknorr) [reply xml/s4s_setup_cluster.xml]: I was hoping toms could include the actual button name here. :)
I think it's just called "Save configuration". (yast2-sap-ha-1.0.5/src/lib/sap_ha/wizard/summary_page.rb:49).

File: xml/s4s_setup_cluster.xml (hunk 2 above)
Comment (diegoakechi) [reply xml/s4s_setup_cluster.xml]: @sknorr  1 & 2: Need to be executed on the production machines, as the module tests the existence of IPs, disks, watchdog, etc. And it also needs to have HANA installed on it. The main idea is that the user do it manually on the first time and save the file. If he needs to reproduce it by any reason, like recreating the environment, then he can use the saved configuration to do it automatically.

3 & 4: Is needed to be executed only on the primary node. The module connects to the secondary via SSH and perform the actions there automatically. The yast module RPM package needs to be installed on both nodes previously.

File: xml/s4s_setup_cluster.xml (hunk 3 above)
Comment (diegoakechi) [reply xml/s4s_setup_cluster.xml]: Correct!

Comment (tomschr) [reply xml/s4s_setup_cluster.xml]: Perfect, I've corrected this sentence to "On the last screen, click the `Save configuration` button."

File: xml/s4s_setup_cluster.xml (hunk 4 above)
Comment (tomschr) [reply xml/s4s_setup_cluster.xml]: As mentioned before, step 2 & 3 are executed on the primary HANA machine. I've added the "On the primary HANA machine..." on step 3, so it should be clear that this step is also executed on the primary.

**Pull request merged**